ONSdigital/rasrm-acceptance-tests
# Edit survey details

# Motivation and Context
To spread the knowledge about parallel tests , modify survey chosen because of its relative independence

# What has changed
Modified test to use data created during fixture execution. Minor import clean up, minor code tweaks in is_valid_parallel_environment and is_valid_sequential_environment. 
Modified Create survey to be parallelizable. Added support for matching to context object attributes in feature then strings. E.g "Then they get an error message of 'Survey with ID {survey_ref} already exists" in create_survey.

REDUCED RANDOMNESS of survey reference from 6 integers to 5 since a 9 was mandated as first and 7 characters fails validation

# How to test?
Make sure acceptance tests pass, view with local ui to validate survey created during test 

# Links
 https://trello.com/c/kxHiqt1C
